package-lock.json

```diff
@@ -0,0 +1,1578 @@
+{
```

Comment:
This file should not exist here. There should only be package-lock.json files in the backend and frontend folders